config/kernel/linux-rockchip-rk3588-legacy.config

```diff
@@ -2880,8 +2896,6 @@ CONFIG_BCMDHD_NVRAM_PATH="/lib/firmware/ap6275p/nvram.txt"
 # CONFIG_BCMDHD_STATIC_IF is not set
 # CONFIG_CYW_BCMDHD is not set
 # CONFIG_INFINEON_DHD is not set
-CONFIG_RTL8852BE=m
-CONFIG_RTL8852BU=m
```

Comment:
Isn't it needed for Radxa A8 WiFi module?

Reply:
There are some drivers added by radxa which I haven't added to this kernel, I will check them later.